craft_providers/lxd/launcher.py

```diff
@@ -630,13 +630,14 @@ def launch(
         return instance
 
     # at this point, there is a valid base instance to be copied to a new instance
-    logger.warning(
-        "Creating instance from base instance %r", base_instance.instance_name
+    logger.info("Creating instance from base instance")
+    logger.debug(
+        "Creating instance from base instance %r.", base_instance.instance_name
     )
 
     # the base instance is not expected to be running but check for safety
     if base_instance.is_running():
-        logger.warning("Stopping base instance.")
+        logger.debug("Stopping base instance.")
```

Comment:
@mr-cal while trying to cover this line I realized that we log that we're stopping the base instance but we don't actually call `base_instance.stop()` here. Is that right?

Reply:
Good catch.  I think you're right - the base instance should be stopped here.

On a related note, https://github.com/canonical/craft-providers/pull/376 is supposed going to flip the order of creation (previously: creating an instance and copying to the base instance new: create a base instance and copy it to the instance).

I'll keep this in mind when I review that PR.  If this is still outstanding, I'll make an issue (or just fix it).

Reply:
here's what I propose: I don't think we should make that kind of change in this "logging" PR, so I'll add the test checking that `base_instance.stop()` is _not_ called. Then when we get around to fixing it we have an existing, breaking test that just needs to be updated. Is that ok?

Reply:
Yes!

Reply:
I think you can just leave it there for now, I will rebase my PR on top of that. So you don't need to keep tracking what I am changing.

Reply:
thanks!